src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -80,5 +80,5 @@ abstract contract StrategyManagerStorage is IStrategyManager {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[40] private __gap;
+    uint256[42] private __gap;
```

Comment:
is this fixing a previous error in the storage gap size? not seeing it paired with any other storage changes?

Reply:
same thing, added storage, updated the gap and then deleted said storage.  big oof on my part, fixed